libr/util/graph_drawable.c

```diff
@@ -0,0 +1,100 @@
+#include <r_core.h>
+#include <r_util/r_graph_drawable.h>
+
+R_API void r_graph_free_node_info(void *ptr) {
+	if (!ptr) {
+		return;
+	}
+	RGraphNodeInfo *info = ptr;
+	free (info->body);
+	free (info->title);
+	free (info);
+}
+
+R_API RGraphNodeInfo *r_graph_create_node_info(const char *title, const char *body, ut64 offset) {
+	RGraphNodeInfo *data = R_NEW0 (RGraphNodeInfo);
+	if (data) {
+		data->title = R_STR_DUP (title);
+		data->body = R_STR_DUP (body);
+		data->offset = offset;
+	}
+	return data;
+}
+
+R_API RGraphNode *r_graph_add_node_info(RGraph *graph, const char *title, const char *body, ut64 offset) {
+	r_return_val_if_fail (graph, NULL);
+	RGraphNodeInfo *data = r_graph_create_node_info (title, body, offset);
+	if (!data) {
+		return NULL;
+	}
+	return r_graph_add_nodef (graph, data, r_graph_free_node_info);
+}
+
+char *r_graph_drawable_to_dot (RGraph /*RGraphNodeInfo*/ *graph, const char *node_properties, const char *edge_properties) {
```

Comment:
If its public use R_API

Also im not 100% happy eith the drawable name but im fine for all the changes proposed here

Reply:
Do you have suggestions for better name?

Reply:
Adding missing R_API to function definitions.